cypress-shared/support/common/testcase.js

```diff
@@ -16,35 +18,41 @@ export function configureTargetWorkspace(app, version, workspace = 'master') {
     updateRetry(2),
     () => {
       cy.getVtexItems().then((vtex) => {
```

Comment:
At some point we need to take down this call... we have the `vtex` already on `config`.